style.css

```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
 ul, li{
     list-style: none;
 }
+
 .logo_header{
-    margin-top: 30px;
+    padding-top: 41px;
     margin-left: 184px;
+    padding-right: 123px;
 }
-.wrapper{
+.header__inner{
     display: flex;
+    justify-content: space-around;
+}
+.nav__link{
+    padding-top: 44px;
+    margin-left: 46px;
+    text-decoration: none;
+    color: #696871;
+    font-family:'DM Sans', sans-serif;
+    font-size: 0.9375rem;
+    line-height: 19px;
+}
+.nav__link:hover{
+    color:#19191B ;
+    text-decoration:underline;
+}
+.nav{
+    padding-top: 44px;
+}
+.btn__header{
+    padding-top:30px ;
+    margin-left: 159px;
+    margin-right: 180px;
+}
+.btn__inner{
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    background-color: #FF7143;
+    color:white ;
+    border-radius: 10px;
+    padding: 19px 60px 19px 60px;
+    border: none;
+    letter-spacing: -0.5px;
+    width: 211px;
+}
+.container__hero{
+    padding-top: 100px;
+    margin-left: 184px;
+}
+.hero__inner{
+    display: flex;
+}
+.hero__sub{
+    width: 459px;
+    margin-right: 77px;
+}
+.hero_title{
+    font-size: 5rem;
+    line-height: 77px;
+    letter-spacing: -2.50105px;
+    color: #19191B;
+    padding-top: 92px;
+}
+.hero__text{
+    font-family:'Mulish', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    letter-spacing: -0.625263px;
+    color: #696871;
+}
+.hero_btn{
+    margin-top: 50px;
+    
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: white;
```

Comment:
you should write color in one style
if write color as HEX, write it

Reply:
done